### ISSUE TITLE
connection failure occurs on the network using the proxy

### ISSUE THREAD

**Issue by earth55** [Azure-Samples/cognitive-services-speech-sdk#5]
This app runs well on the network that don't use proxy.
However, this application outputs a 'connection error' on the network using the proxy.
I want to run this application on a network using a proxy.

### This issue is for a: (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [x] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> Run application, select the option 1

### Any log messages given by the failure
```
 1. Speech recognition with microphone input.
 2. Speech recognition in the specified language.
 3. Speech recognition with file input.
 4. Speech recognition using customized model.
 5. Speech continuous recognition using events.
 6. Translation with microphone input.
 7. Translation with file input.
 Your choice (0: Stop.): 1
 Say something...
 There was an error. Status:Canceled, Reason:Connection failed (no connection to the remote host).
 
 Recognition done. Your Choice (0: Stop):
```

### Expected/desired behavior
> No Connection failed occurs.

### OS and Version?
> Windows 8.1 Pro

### Versions
> cognitive-services-speech-sdk/Windows/csharp_samples/csharp_samples.sln (042626c : Commits on May 25, 20)

### Mention any other details that might be useful
> none

**Comment by Thenmozhi16** [comment]
I am also facing this issue in proxy network. Can you please update when the fix will be available

**Comment by FrColin** [comment]
I am also facing this issue in proxy network. october News ????

**Comment by wolfma61** [comment]
@FrColin, @Thenmozhi16, @earth55 
Please take a look at the just released version 1.1.0 if this fixes your issues.
https://docs.microsoft.com/azure/cognitive-services/speech-service/releasenotes#speech-service-sdk-110

**Comment by wolfma61** [comment]
I'm closing this - if you have further questions regarding proxy support, please open a new issue

thx
